packages/preview-middleware-client/src/adp/controllers/AddSubpage.controller.ts

```diff
@@ -153,16 +141,16 @@ export default class AddSubpage extends BaseDialog<AddSubpageModel> {
             };
         } else {
             const routePattern = generateRoutePattern(
-                this.model.getProperty('/routePattern'),
+                this.options.pageDescriptor.routePattern ?? '',
```

Comment:
`routePattern` should always be defined, no need to use the nullish coalescing operator.

Reply:
fixed